org.zaproxy.ZAP.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop-application">
+  <id>org.zaproxy.ZAP</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>ZAP</name>
+  <summary>
+    The OWASP Zed Attack Proxy (ZAP) is a popular free security 
+    tool.
+  </summary>
+  <description>
+    <p>
+      The OWASP Zed Attack Proxy (ZAP) is one of the world’s most popular 
+      free security tools and is actively maintained by hundreds of international 
+      volunteers. It can help you automatically find security vulnerabilities 
+      in your web applications while you are developing and testing your 
+      applications. Its also a great tool for experienced pentesters to use for 
+      manual security testing.
+    </p>
+    <p>
+      NOTE: This wrapper is not verified by, affiliated with, or supported by
```

Comment:
No need for such disclaimer given it's open source and upstream team is happy to have it on Flathub.

Reply:
sweet.